include/Macro.h

```diff
@@ -654,6 +654,7 @@
 #ifdef GRAVITY
 #  define EXT_POT_NAUX_MAX       20    // ExtPot_AuxArray[]
 #  define EXT_ACC_NAUX_MAX       20    // ExtAcc_AuxArray[]
+#  define EXT_GENEPTR_NAUX_MAX    6    // h/d_GenePtrArray
```

Comment:
How about renaming it as `EXT_POT_NGENE_MAX` to better distinguish it from `*_NAUX_*`, which is for `*AuxArray*` arrays?